src/components/MultiSearchInput/index.tsx

```diff
@@ -22,6 +22,15 @@ export interface MultiSearchInputProps {
   noOptionsPlaceholder?: string;
 }
 
+interface displayValue {
```

Comment:
this interface is not called displayvalue, it's actually the DropdownData, also you wouldn't include `length` in its interface